lib/elixir/lib/base.ex

```diff
@@ -150,8 +150,10 @@ defmodule Base do
   @doc """
   Encodes a binary string into a base 16 encoded string.
 
-  Accepts an atom `:upper` (default) for encoding to upper case characters or
-  `:lower` for lower case characters.
+  ## Options
```

Comment:
Maybe we should reuse the format from the https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/file.ex#L257-L266?
To me it seems easier to read.